.circleci/docker/build.sh

```diff
@@ -111,21 +111,9 @@ case "$image" in
     DB=yes
     VISION=yes
     ;;
-  pytorch-linux-xenial-cuda10.2-cudnn7-py3-gcc7)
```

Comment:
Please original cuda 10.2, cuda 11.3 and cuda 11.5 workflows we need a new one for 11.6